File: Raspberry Pi with Python/GBHmission01.py
import RPi.GPIO as GPIO
import time

LEDRED = 4
LEDGREEN = 5
LEDBLUE = 6

GPIO.setmode(GPIO.BCM)
GPIO.setup(4, GPIO.OUT)
GPIO.setup(5, GPIO.OUT)
GPIO.setup(6, GPIO.OUT)

for i in range(0, 10):
    GPIO.output(4, True)
    time.sleep(0.2)
    GPIO.output(4, False)
    GPIO.output(5, True)
    time.sleep(0.2)
    GPIO.output(5, False)
    GPIO.output(6, True)
    time.sleep(0.2)
    GPIO.output(6, False)

GPIO.cleanup()

File: Raspberry Pi with Python/GBHmission01.c
#include <stdio.h>
#include <wiringPi.h>

#define LEDRED 7
#define LEDGREEN 21
#define LEDBLUE 22

int main(void)
{
    if (wiringPiSetup() == -1)
		return 1;

    pinMode(LEDRED, OUTPUT);
    pinMode(LEDGREEN, OUTPUT);
    pinMode(LEDBLUE, OUTPUT);
    digitalWrite(LEDRED, 0);
    digitalWrite(LEDGREEN, 0);
    digitalWrite(LEDBLUE, 0);

    for (int i = 0; i < 10; i++)
    {
        digitalWrite(LEDRED, 1);
        delay(200);
        digitalWrite(LEDRED, 0);
        digitalWrite(LEDGREEN, 1);
        delay(200);
        digitalWrite(LEDGREEN, 0);
        digitalWrite(LEDBLUE, 1);
        delay(200);
        digitalWrite(LEDBLUE, 0);
    }

    return 0;
}